File: for_loop/ex1.cpp
#include<iostream>
using namespace std;

bool findNum(int myArr[], int num);
int main() {
    int myArr[5] = {2,10,9,8,7};
    int num = 5;
    bool found = findNum(myArr, num);

    if (found) {
        cout << "Find the number" << endl;
    } else {
        cout << "Cannot find the number" << endl;
    }

    return 0;
}

/** Find whether an number in the given array */
bool findNum (int myArr[], int num) {
    for(int pos = 0; pos < 5; pos++) {
        if(num == myArr[pos]) {
            return true;
        }
    }
    return false;
}

File: recursive/ex1.py
# find the factorial of an number

def findFactorial(inputNum): 
    if inputNum == 1 or inputNum == 0: 
        return 1
    
    return inputNum * findFactorial(inputNum - 1)


inputNum = 10
factorial = findFactorial(inputNum);
print(f"The factorial of {inputNum} is {factorial}.")